From 47469bf36d87cd3c144e415c41381403044341ee Mon Sep 17 00:00:00 2001
From: zachary <zhangzg@marvell.com>
Date: Tue, 28 Mar 2017 17:48:36 +0800
Subject: [PATCH 0089/1239] arm64: mvebu: config: a37xx: enable pca953x

There is PCA953x io-expander on db board.
This patch enables io-expander as gpio function.

Change-Id: I4af9b459c55f906793ddcc0f5060bee8ae76987c
Signed-off-by: zachary <zhangzg@marvell.com>
---
 configs/mvebu_db-88f3720_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mvebu_db-88f3720_defconfig b/configs/mvebu_db-88f3720_defconfig
index 1da70ea94c..286f72a5b7 100644
--- a/configs/mvebu_db-88f3720_defconfig
+++ b/configs/mvebu_db-88f3720_defconfig
@@ -43,6 +43,7 @@ CONFIG_CLK=y
 CONFIG_CLK_MVEBU=y
 CONFIG_DM_GPIO=y
 # CONFIG_MVEBU_GPIO is not set
+CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
 CONFIG_MISC=y
 CONFIG_DM_MMC=y
-- 
2.29.0

